lib/fulcrum_extract_gravity_freshness.rb

```diff
@@ -0,0 +1,65 @@
+class DataFreshness
+  S3_LOCATIONS = {
+    fair_organizers: { bucket: 'artsy-data', prefix: 'reports/gravity_fair_organizers/' },
+    artist_series_artworks: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_subscription_plans/' },
+    merchant_accounts: { bucket: 'artsy-data', prefix: 'reports/gravity_merchant_accounts/' },
+    sales: { bucket: 'artsy-data', prefix: 'reports/gravity_sales/' },
+    fairs: { bucket: 'artsy-data', prefix: 'reports/gravity_fairs/' },
+    profiles: { bucket: 'artsy-data', prefix: 'reports/gravity_profiles/' },
+    representatives: { bucket: 'artsy-data', prefix: 'reports/gravity_representatives/' },
+    partner_subscriptions: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_subscriptions/' },
+    partner_locations: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_locations/' },
+    partners: { bucket: 'artsy-data', prefix: 'reports/gravity_partners/' },
+    purchases: { bucket: 'artsy-data', prefix: 'reports/gravity_purchases' },
+    account_requests: { bucket: 'artsy-data', prefix: 'reports/gravity_account_requests' },
+    partner_subscription_charges: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_subscription_charges' },
+    bidders: { bucket: 'artsy-data', prefix: 'reports/gravity_bidders' },
+    credit_cards: { bucket: 'artsy-data', prefix: 'reports/gravity_credit_cards' },
+    sale_artworks: { bucket: 'artsy-data', prefix: 'reports/gravity_sale_artworks' },
+    bidder_positions: { bucket: 'artsy-data', prefix: 'reports/gravity_bidder_positions' },
+    offer_emails: { bucket: 'artsy-data', prefix: 'reports/gravity_offer_emails' },
+    user_interests: { bucket: 'artsy-data', prefix: 'reports/gravity_user_interests' },
+    bids: { bucket: 'artsy-data', prefix: 'reports/gravity_bids' },
+    partner_subscription_charge_line_items: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_subscription_charge_line_items' },
+    offer_actions: { bucket: 'artsy-data', prefix: 'reports/gravity_offer_actions' },
+    partner_shows: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_shows' },
+    partner_artists: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_artists' },
+    partner_show_artworks: { bucket: 'artsy-data', prefix: 'reports/gravity_partner_show_artworks' },
+    artists: { bucket: 'artsy-data', prefix: 'reports/gravity_artists' },
+    inquiry_requests: { bucket: 'artsy-data', prefix: 'reports/gravity_inquiry_requests' },
+    collector_profiles: { bucket: 'artsy-data', prefix: 'reports/gravity_collector_profiles' },
+    edition_sets: { bucket: 'artsy-data', prefix: 'reports/gravity_edition_sets' },
+    users: { bucket: 'artsy-data', prefix: 'reports/gravity_users' },
+    follow_artists: { bucket: 'artsy-data', prefix: 'reports/gravity_follow_artists' },
+    artwork_versions: { bucket: 'artsy-data', prefix: 'reports/gravity_artwork_versions' },
+    artworks: { bucket: 'artsy-data', prefix: 'reports/gravity_artworks' },
```

Comment:
I think the prefixes are actually different, like `reports/gravity.artworks/`. Also, the trailing slash is important so that a path like `...artist_series` doesn't match others like `...artist_series_artworks`.